test/check-machines-create

```diff
@@ -81,6 +81,17 @@ class TestMachinesCreate(VirtualMachinesCase):
                                                             start_vm=False,
                                                             pixel_test_tag="auto"))
 
+        # Test OS auto detection can change "Operatin system" field without Ooops
```

Comment:
s/Operatin/Operating

Reply:
Do you have some bugzilla to attach in the test as comment? No needed just wondering.

Reply:
Yeah, I missed it, this case is related with https://bugzilla.redhat.com/show_bug.cgi?id=1715409